stackslib/src/config/mod.rs

```diff
@@ -2824,13 +2826,15 @@ pub struct EventObserverConfigFile {
     pub endpoint: String,
     pub events_keys: Vec<String>,
     pub timeout_ms: Option<u64>,
+    pub lossy: Option<bool>,
```

Comment:
Not trying to bike-shed here, but I recall it said that the name of this field should be sufficiently verbose that no one would accidentally turn it on.  Is `lossy` sufficiently expressive?  Maybe `unreliable_delivery` would be a more explicit name?  Just want to make sure @hstove and @obycode are aware.

Reply:
Yep, we briefly talked about it in the naka meeting, but we can definitely "improve" it. ```unreliable_delivery``` looks good to me btw.

Reply:
I was happy with the level of peril that `lossy` indicates 😅, but I could get behind something more scary. Other alternatives that come to mind are `drop_on_failure` or `disable_retries`, as they are more explicit in what it means.

Reply:
From meeting -- et's go with `disable_retries`.